docs/content/Schema/Getting-Started.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Getting Started
+permalink: /schema/getting-started
+category: Reference
+menuOrder: 1
+proofread: 06/18/2019
+redirect_from:
+  - /getting-started-cubejs-schema
+---
+
+A Cube.js Data Schema is used to model raw data into meaningful business
+definitions and pre-aggregate and optimize results. The data schema is exposed
```

Comment:
Maybe "...and pre-aggregate data for optimal results"? The two "and's" sound a bit awkward.